python/mathsys/main/parser.py

```
@@ -1,122 +1,122 @@
+from lark import Lark, Transformer, Token
+
+
+#
+#   1ºLEVEL
+#
+
+# 1ºLEVEL -> SHEET
+@dataclass
+class Sheet:
+    type: [
+        "empty" | "inline" | "normal"
```

Ruff (0.12.2)

Undefined name `empty`

(F821)